crates/typst-library/src/visualize/image.rs

```diff
@@ -64,6 +68,60 @@ pub struct ImageElem {
     pub fit: ImageFit,
 }
 
+/// A raster or vector graphic loaded from string.
+///
+/// Supported formats are PNG, JPEG, GIF and SVG.
+///
+/// _Note:_ Work on SVG export is ongoing and there might be visual inaccuracies
+/// in the resulting PDF. Make sure to double-check embedded SVG images. If you
+/// have an issue, also feel free to report it on [GitHub][gh-svg].
```

Comment:
No need to repeat this. Both the element and this function will end up on the same documentation page.